### ISSUE TITLE
500 Error while deleting a comment

### ISSUE THREAD

**Issue by mkalam-alami** [alakajam-team/alakajam#260]
It was correctly deleted though.

```
[[31m10:31:02.778 ERROR (/core/middleware.js:310)^[[39m Something went wrong, sorry about that.
TypeError: Cannot read property 'split' of undefined
    at Object.handleSaveComment (/var/www/alakajam/controllers/post-controller.js:411:9)
    at <anonymous>
```

## Bonus

500 errors happening at unknown times (found in the logs):

```
^[[31m02:11:42.615 ERROR (/core/middleware.js:310)^[[39m Something went wrong, sorry about that.
TypeError: Cannot read property 'get' of undefined
    at editEntry (/var/www/alakajam/controllers/entry-controller.js:249:101)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```

```
  <rejected> TypeError: Cannot read property 'get' of undefined
    at Object.refreshEventCounts (/var/www/alakajam/services/event-service.js:855:29)
    at editEntry (/var/www/alakajam/controllers/entry-controller.js:321:22)
    at <anonymous> }
```

```
^[[31m02:12:31.145 ERROR (/server.js:93)^[[39m Unhandled promise rejection: Promise {
  <rejected> Error: Undefined binding(s) detected when compiling SELECT query: select count(*), "division" from "entry" where "event_id" = ? and "published_at" <= ? group by "division"
.... (useless stack trace) 
```

Also, a verbose `console.log` call seems to remain somewhere.

**Comment by coxm** [comment]
I think https://github.com/alakajam-team/alakajam/commit/e16d8a5cefae0922bd67ff5259f4c7a727428afb and https://github.com/alakajam-team/alakajam/commit/937324d74e7f39a6a65dd4e3f971e66910d4d7be fix the first two of these. I suspect the third was fixed by 616a7739.
As for the fourth...

**Comment by coxm** [comment]
Actually on closer inspection I'm pretty sure the fourth was fixed by 616a7739 too ^^.
